site/content/contribute/webapp/redux/actions.md

```diff
@@ -1,38 +1,38 @@
 ---
 title: "Actions"
-heading: "Actions in Redux - Mattermost"
-description: "Similar to other frameworks like Flux, actions in Redux represent a single change to the Redux store as a plain JavaScript object."
+heading: "Redux Actions"
+description: "An explanation of Redux actions and how they're used in Mattermost"
 date: 2017-08-20T11:35:32-04:00
 weight: 4
 ---
 
-Similar to other frameworks like Flux, actions in Redux represent a single change to the Redux store as a plain JavaScript object.
+Actions in Redux represent an operation either performed by the user or the server which will cause a change to the state of the web app which is stored in the Redux store. It is generally represented as a plain JavaScript object with a constant `type` string and often other data stored in fields such as `data`.
 
-```javascript
+```typescript
 {
     type: 'SELECT_CHANNEL',
-    data: channelId
+    data: channelId,
 }
 ```
 
 They are created by functions called action creators. In regular Redux, this function will take some arguments and return an action representing how the store should be changed. Something to note with Mattermost Redux is that we typically refer to the action creators as the "actions" themselves since there's often a single action creator for a given type of action.
```

Comment:
```suggestion
Actions are created by functions called action creators. In regular Redux, this function will take some arguments and return an action representing how the store should be changed. Something to note with Mattermost Redux is that we typically refer to the action creators as the "actions" themselves since there's often a single action creator for a given type of action.
```